docs/pages/getting-started/4-connecting-crm.rst

```diff
@@ -0,0 +1,216 @@
+If all is well, the node will respond with a DID Document similar to:
+
+.. code-block:: json
+
+    {
+      "@context": [ "https://www.w3.org/ns/did/v1" ],
+      "id": "did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9",
+      "verificationMethod": [
+        {
+          "id": "did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9#_TKzHv2jFIyvdTGF1Dsgwngfdg3SH6TpDv0Ta1aOEkw",
+          "controller": "did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9",
+          "type": "JsonWebKey2020",
+          "publicKeyJwk": {
+            "crv": "P-256",
+            "x": "38M1FDts7Oea7urmseiugGW7tWc3mLpJh6rKe7xINZ8",
+            "y": "nDQW6XZ7b_u2Sy9slofYLlG03sOEoug3I0aAPQ0exs4",
+            "kty": "EC"
+          }
+        }],
+      "capabilityInvocation": [
+        "did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9#_TKzHv2jFIyvdTGF1Dsgwngfdg3SH6TpDv0Ta1aOEkw"
+      ],
+      "assertion": [
+        "did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9#_TKzHv2jFIyvdTGF1Dsgwngfdg3SH6TpDv0Ta1aOEkw"
+      ],
+      "service": []
+    }
+
+The ``id`` at the top level needs to be extracted and stored as your vendor DID.
+In the example above this would be ``did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9``
+
+Setting vendor contact information
+==================================
+
+Things can go wrong: a node is misbehaving or a DID Document is conflicted.
+If the node operator is not resolving the problem it's extremely convenient if others can contact the node operator and relay the problem.
+For this use-case, Nuts supports the registration of node contact information. The contact information will be added to a DID Document as a service.
+A convenience API is available to add the contact information to a DID Document. The vendor DID should be used for this.
+
+.. code-block:: text
+
+    PUT <internal-node-address>/internal/didman/v1/did/{did}/contactinfo
+    {
+        "name": "vendor X",
+        "phone": "06-12345678",
+        "email": "info@example.com",
+        "website": "https://example.com"
+    }
+
+Where ``{did}`` must be replaced with the vendor DID.
+
+Adding endpoints
+================
+
+As a vendor you'll probably be hosting different services at various stages. A Nuts node API is available to easily add/remove the endpoints for these services.
+Registering services is a required step since the services that will be registered for organizations will make use of these services.
+
+.. code-block:: text
+
+    POST <internal-node-address>/internal/didman/v1/did/{did}/endpoint
+    {
+        "type": "example-production-api",
+        "endpoint": "https://api.example.com"
+    }
+
+Where ``{did}`` must be replaced with the vendor DID. The ``type`` may be freely chosen and is used as reference in the organization services.
```

Comment:
why must endpoint be a fully qualified domain name? It entirely depends on the type of endpoint?

Reply:
changed it